index.html

```diff
@@ -274,7 +284,7 @@ <h4>Inclusive Cities Co-Design</h4>
                                 </p>
                                 <p><a href="./news/2018-08-30-Cities-Co-Design.html">Read more: Inclusive Cities Co-Design</a></p>
 
-                                <h4>El Planeta Es La Escuela Workshops</h4>
+                                <h4><a href="./news/2018-06-19-El-Planeta-Es-La-Escuela.html">El Planeta Es La Escuela Workshops</a></h4>
```

Comment:
In Spanish, titles are not capitalized the same way as in English. Only first letter and proper pronouns are capitalized. Everything else is lower case.